analytic_engine/src/meta/mod.rs

```diff
@@ -9,9 +9,20 @@ pub mod meta_update;
 use std::{fmt, sync::Arc};
 
 use async_trait::async_trait;
-use wal::manager::WalLocation;
-
-use crate::meta::{meta_data::TableManifestData, meta_update::MetaUpdateRequest};
+use common_types::table::ShardId;
+use table_engine::table::TableId;
+
+use crate::{
+    meta::{meta_data::TableManifestData, meta_update::MetaUpdateRequest},
+    space::SpaceId,
+};
+
+pub struct LoadRequest {
+    pub space_id: SpaceId,
+    pub table_id: TableId,
+    pub cluster_version: u64,
```

Comment:
this name is not consistent with WalLocation

```
pub struct WalLocation {
    pub versioned_region_id: VersionedRegionId,
    pub table_id: TableId,
}

```

Reply:
Maybe we should keep the original name of the field, because `Manifest` trait doesn't expose anything relevant with WAL.